experiments/standalone/Bucket/global_bucket_staticmap.jl

```diff
@@ -347,5 +349,15 @@ for (i, (field_ts, field_name)) in enumerate(
     end
 
 end
-outfile = joinpath(outdir, string("ts_$device_suffix.png"))
-CairoMakie.save(outfile, fig_ts)
+
+if device_suffix == "cpu"
```

Comment:
I made this to see how gridd-y the field are. I think it's useful to keep?

Reply:
I would prefer not having `Plots` in the dependency tree, it often leads to problems. There should be a way to do this with Makie, but I don't know it

Reply:
it's a long term goal to move our plotting to Makie, but I believe we use plots in all the fluxnet runs also. so perhaps a larger issue.

that said, I can use ClimaCoreMakie here instead!